Model/Contexts/PlaylistDataContext.cs

```diff
@@ -1,27 +1,48 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
 using System.Text;
+using Model.Annotations;
+using Model.DbModels;
 
 namespace Model
 {
-    public class PlaylistDataContext
+    public class PlaylistDataContext : INotifyPropertyChanged
     {
         public string PlaylistName { get; set; }
         public string Description { get; set; }
         public List<SongInfo> PlaylistItems { get; set; }
+        public Playlist Playlist { get; set; }
+        public List<Song> Songs { get; set; }
 
-        public PlaylistDataContext()
+        public void AddPlaylistsToMenu()
         {
-            PlaylistName = "Test playlist";
-            Description = "a description for the test playlist";
+            PlaylistName = Playlist.Name;
```

Comment:
Kan Playlist geen null zijn? Want dan moet je dat wel eerst checken voor je ditt teruggeeft

Reply:
Playlist kan geen null zijn. voordat AddPlayListsToMenu wordt aangeroepen, wordt playlist geset (PlayListView.xaml.cs line 34-36) dat window wordt aangeroepen als er op een playlist geklikt is dus Playlist is nooit null